bugsnag-android-core/src/main/java/com/bugsnag/android/Stackframe.kt

```diff
@@ -7,7 +7,7 @@ class Stackframe internal constructor(
     var file: String?,
     var lineNumber: Number?,
     var inProject: Boolean?,
-    private var customFields: Map<String, Any>
+    private var customFields: Map<String, Any?>
```

Comment:
is the added `?` because the `Any` type does not include `null`?

Reply:
Yep.  `Any` and `Any?` are separate types in Kotlin: the `?` means that it allows null values. There's a good article explaining Kotlin's type hierarchy in a bit more detail here: http://www.natpryce.com/articles/000818.html